app/config.py

```diff
@@ -8,7 +8,7 @@ class Config:
     SECRET_KEY = os.environ.get("SECRET_KEY")
     AUTH_TOKENS = os.environ.get("AUTH_TOKENS")
 
-    DOCUMENTS_BUCKET = None
+    DOCUMENTS_BUCKET = os.getenv("MULTIREGION_ACCESSPOINT_ARN", os.environ.get("DOCUMENTS_BUCKET"))
```

Comment:
`DOCUMENTS_BUCKET` isn't an environment variable that exists, but we can probably get away without the fallback option anyway

Reply:
Yeah my intention was a fallback variable to override it. MULTIREGION_ACCESSPOINT_ARN didn't seem to fit the pattern of environment variable matching property name so I put it in.

I'll remove it if you want. Let me know your thoughts.

Reply:
I see what you mean about the names not matching now. I'm fine with renaming it, I think we just would need to make sure it's really obvious that the value here is not a bucket name (unlike all the other config for buckets) and that it comes from a Terraform output and not an environment variable we hardcode

Reply:
If we wanted to rename it we'd have to do it in two PRs.... We'd need to add both variations to the terraform 

https://github.com/alphagov/notifications-aws/blob/997cf36859843219575cca675d4a83c59da720a0/terraform/notify-apps/service_document_download_api.tf#L72

and then remove the MULTIREGION_ACCESSPOINT_ARN once this change has been merged and rolled out. Do you want to do this as part of this PR?

Reply:
I don't think you need to rename anything (unless you want to) I think I was just a bit confused about what was changing here with the new `DOCUMENTS_BUCKET` environment variable, but I think it's all fine